server/szurubooru/func/posts.py

```diff
@@ -399,7 +399,10 @@ def _after_post_update(
 def _before_post_delete(
         _mapper: Any, _connection: Any, post: model.Post) -> None:
     if post.post_id:
-        image_hash.delete_image(post.post_id)
+        image_hash.delete_image(str(post.post_id))
```

Comment:
Why `str()` ?

Reply:
I think that was because I was reading IDE comments and image_hash.delete_image accepts a string according to the typing hints. That shouldn't have made it into the commit as it is not needed.

```python
@_safety_blanket(lambda: None)
def delete_image(path: str) -> None:
    assert path
...
```